controllers/ApiController.php

```diff
@@ -56,7 +56,7 @@ public function behaviors()
                 'upload' => ['POST'],
                 'resolve-permissions' => ['POST'],
                 'change-permissions' => ['POST'],
-                'search' => ['GET']
+                'search' => ['GET', 'POST']
```

Comment:
Why is `POST` allowed for `search`? It actually doesn't change anything on the server.

Reply:
Right, doesn't change anything. But using POST for such requests is valid, it would only be a problem the other way round, i.e. if we were to allow GET for non-idempotent requests.

By the way: for the `list` action, which also writes nothing, only POST is allowed ;-)